jormungandr-lib/src/interfaces/reward_parameters.rs

```diff
@@ -11,12 +11,18 @@ pub enum RewardParams {
         ratio: Ratio,
         epoch_start: Epoch,
         epoch_rate: NonZeroU32,
+        rewards_limit: Ratio,
+        npools: NonZeroU32,
+        npools_threshold: NonZeroU32,
```

Comment:
the parameters are not very explicit. `npools` is very vague.

Reply:
They're described in the comments and will be discussed in blogs etc.  Other suggestions for names are welcome (but be aware that this may affect the code as well, of course).